core/include/detray/geometry/shapes/unbounded.hpp

```diff
@@ -32,8 +34,11 @@ class unbounded {
     template <typename scalar_t>
     using bounds_type = darray<scalar_t, boundaries::e_size>;
 
+    /// Convenience member to construct the name
+    static constexpr std::string_view __prefix = "unbounded ";
```

Comment:
Isn't a double underscore expected for e.g. a standard library kind of variable naming? Should we really use this convention anywhere in detray?

Reply:
Fair, I will just rename it to `prefix`.